### PR TITLE
chore: Fix formatting after black; run static analysis tools

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
# Submitting Pull Requests

Hi @pennersr ,
This PR fixes wrong black formatting which were introduced in [next commit](https://github.com/pennersr/django-allauth/commit/da5ccdcf171e32ab1a438add3af38957f5a0659a).

## General

 - [x] Make sure you use [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages).
       Examples: `"fix(google): Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
 - [x] All Python code must formatted using Black, and clean from pep8 and isort issues.
 - [ ] JavaScript code should adhere to [StandardJS](https://standardjs.com).
 - [ ] If your changes are significant, please update `ChangeLog.rst`.
 - [ ] If your change is substantial, feel free to add yourself to `AUTHORS`.

 ## Provider Specifics

 In case you add a new provider:

- [ ] Make sure unit tests are available.
- [ ] Add an entry of your provider in `test_settings.py::INSTALLED_APPS` and `docs/installation.rst::INSTALLED_APPS`.
- [ ] Add documentation to `docs/providers.rst`.
- [ ] Add an entry to the list of supported providers over at `docs/overview.rst`.

### CODE REVIEW

**Review by yasinByasin94 — APPROVED** [review]

File: allauth/account/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import models, migrations
 import django.utils.timezone
 from django.conf import settings
+from django.db import migrations, models
+
```
Comment (yasinByasin94) [inline allauth/account/migrations/0001_initial.py:7]: remove useless lines

File: allauth/account/utils.py [hunk 2]
```diff
@@ -357,7 +357,7 @@ def send_email_confirmation(request, user, signup=False, email=None):
             get_adapter(request).add_message(
                 request,
                 messages.INFO,
-                "account/messages/" "email_confirmation_sent.txt",
+                "account/messages/email_confirmation_sent.txt",
```
Comment (yasinByasin94) [inline allauth/account/utils.py:360]: use single cotation  ( ' ) instead of duble ( " )

File: allauth/account/views.py [hunk 3]
```diff
@@ -428,7 +428,7 @@ def form_valid(self, form):
         get_adapter(self.request).add_message(
             self.request,
             messages.INFO,
-            "account/messages/" "email_confirmation_sent.txt",
+            "account/messages/email_confirmation_sent.txt",
```
Comment (yasinByasin94) [inline allauth/account/views.py:431]: use single cotation ( ' ) instead of duble ( " )

File: allauth/account/migrations/0001_initial.py (hunk 1 above)
Comment (DmytroLitvinov) [reply allauth/account/migrations/0001_initial.py:7]: @yasinByasin94 , could you explain what do you mean under "useless lines"?
If you check the [whole file](https://github.com/pennersr/django-allauth/blob/13a733edd328b75619799bf09de5b7c4310ad54a/allauth/account/migrations/0001_initial.py) you will see that these imports are used.

File: allauth/account/utils.py (hunk 2 above)
Comment (DmytroLitvinov) [reply allauth/account/utils.py:360]: @yasinByasin94 , in that package `black` is used.

File: allauth/account/migrations/0001_initial.py (hunk 1 above)
Comment (sicin) [reply allauth/account/migrations/0001_initial.py:7]: @DmytroLitvinov he probably meant that you added an empty line underneath imports

File: allauth/account/views.py (hunk 3 above)
Comment (DmytroLitvinov) [reply allauth/account/views.py:431]: @yasinByasin94 , in that package `black` is used.

File: allauth/account/migrations/0001_initial.py (hunk 1 above)
Comment (DmytroLitvinov) [reply allauth/account/migrations/0001_initial.py:7]: @sicin, this is how `isort` formatted it.

**Comment by pennersr** [comment]
Closing -- outdated

**Comment by DmytroLitvinov** [comment]
Hi @pennersr ,
I am ready to rebase changes.
Let me know if you are ready to review and merge it.

**Comment by pennersr** [comment]
I am not sure what the intention is for this PR. The code is already black formatted.

**Comment by DmytroLitvinov** [comment]
@pennersr , have you had a chance to check the description of PR and changes?

After reformating with `black` there are gaps in the concatenation of strings. This PR fixes that.

**Comment by DmytroLitvinov** [comment]
Selection_1103

**Comment by pennersr** [comment]
@DmytroLitvinov I see now -- would be happy to merge a rebased PR.

**Comment by DmytroLitvinov** [comment]
@pennersr , rebased. 
Could you please recheck and provide your feedback?

**Comment by DmytroLitvinov** [comment]
Ping @pennersr in case you missed it.

**Pull request merged**